storybook/webpack.config.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
             loader: 'babel-loader',
             options: {
               rootMode: 'upward',
+              presets: [['react-app', { typescript: true }]],
```

Comment:
This line makes [the integration test fail](https://circleci.com/gh/algolia/react-instantsearch/17568). The `react-app` presets throw an error if `NODE_ENV` is `undefined`. The integration test does not explicitly set it so we have to update the command to set `NODE_ENV` to `development`. I've tried to catch the error at the script level but it seems that Storybook swallow it.